lib/Cake/Controller/Component/RequestHandlerComponent.php

```diff
@@ -505,7 +505,12 @@ public function accepts($type = null) {
  *   in the request content type will be returned.
  */
 	public function requestedWith($type = null) {
-		if (!$this->request->is('post') && !$this->request->is('put') && !$this->request->is('delete')) {
+		if (
+			!$this->request->is('patch') &&
```

Comment:
It would be good to get a test to ensure requestedWith() continues to work this way in the future.